Library/Formula/mariadb-connector-c.rb

```diff
@@ -0,0 +1,54 @@
+class MariadbClientRequirement < Requirement
+  fatal true
+  default_formula "mariadb"
+  satisfy { which "mysql" }
+end
+
+class MariadbConnectorC < Formula
+  desc "MySQL & MariaDB database connector for C/C++ applications"
+  homepage "https://github.com/MariaDB/mariadb-connector-c"
+  stable do
+    url "https://github.com/MariaDB/mariadb-connector-c.git",
+        :revision => "330b7fbdbabad44180862316d8925b302d1a8409"
```

Comment:
@DomT4 These guys don't understand how to use git, release commit: https://github.com/MariaDB/mariadb-connector-c/commit/330b7fbdbabad44180862316d8925b302d1a8409
/cc @9E0R9